packages/admin/admin/src/form/Autocomplete.tsx

```diff
@@ -43,14 +42,18 @@ export const FinalFormAutocomplete = <
     loading = false,
     loadingError,
     isAsync = false,
-    clearable,
+    required,
+    disabled,
+    readOnly,
```

Comment:
Is it correct to extract these props here?

Reply:
Yes, I prefer this, over using values from `rest`